### DISCUSSION TITLE
Как сделать многопользовательскую таблицу?

### DISCUSSION THREAD

**Discussion by aityomin** [totumonline/totum-ru-issues-and-discussions#433] [Как это сделать?] [answered]
Есть довольно большая таблица, более 30 полей.
Ее должны заполнять операторы и внешние пользователи.
Операторов много. Пока один вносит данные или правит у других работа останавливается.
А с внешними пользователями вообще беда. Они могут работать каждый только со своей строкой (но это как раз не проблема), главное, что вид таблицы должен быть другим (в виде интерактивной формы из хедеров и секций).
Вроде это можно сделать временными таблицами, но не могу понять как...

**Comment by ttm-alexeypolunin** [comment, marked as answer]
А что значит работа останавливается?

**PS: по архитектуре решений** — записывайтесь на [платную консультацию/сопровождение проекта](https://ru.totum.online/learning). Здесь и в telegram невозможно на такие вопросы ответить.

**Reply by aityomin** [reply]
Значит, что с таблицей невозможно работать. Что в формате таблицы, что в формате панели начинает курсор прыгать по ячейкам, вводить данные не получается.

**Reply by ttm-alexeypolunin** [reply]
Ранее никто не жаловался - если найдете причину, расскажите - поправим.
